child_compassion/wizards/child_search_attributes.py

```diff
@@ -0,0 +1,124 @@
+
+    name = fields.Char()
+    type = fields.Selection(selection=[
+        ('Available', _('Available')),
+        ('Change Commitment Hold', _('Change Commitment Hold')),
+        ('Consignment Hold', _('Consignment Hold')),
+        ('Delinquent Mass Cancel Hold', _('Delinquent Mass Cancel Hold')),
+        ('E-Commerce Hold', _('E-Commerce Hold')),
+        ('Inactive', _('Inactive')),
+        ('Ineligible', _('Ineligible')),
+        ('No Money Hold', _('No Money Hold')),
+        ('Reinstatement Hold', _('Reinstatement Hold')),
+        ('Reservation Hold', _('Reservation Hold')),
+        ('Sponsor Cancel Hold', _('Sponsor Cancel Hold')),
+        ('Sponsored', _('Sponsored')),
+        ('Sub Child Hold', _('Sub Child Hold'))])
+    hold_expiration_date = fields.Datetime()
+    primary_owner = fields.Char()
+    secondary_owner = fields.Char()
+    no_money_yield_rate = fields.Float()
+    yield_rate = fields.Float()
+
+    ##########################################################################
+    #                             FIELDS METHODS                             #
+    ##########################################################################
+
+    @api.multi
+    def send(self):
+
+        holds = self.env['compassion.hold']
+        messages = self.env['gmc.message.pool']
+        global_child = self.env['compassion.childpool.search'].browse(
+            self.env.context.get('active_id'))
+
+        for child in global_child.global_child_ids:
+            # Save children form global children to compassion children
+
+            child_vals = {
+                'global_id': child.global_id,
+                'local_id': child.local_id,
+                'project_id': child.project_id.id,
+                'field_office_id': child.field_office_id,
+                'name': child.name,
+                'firstname': child.firstname,
+                'lastname': child.lastname,
+                'preferred_name': child.preferred_name,
+                'gender': child.gender,
+                'birthdate': child.birthdate,
+                'age': child.age,
+                'is_orphan': child.is_orphan,
+                'beneficiary_state ': child.beneficiary_state,
+                'sponsorship_status': child.sponsorship_status,
+                'unsponsored_since': child.unsponsored_since,
+            }
+            child_comp = self.env['compassion.child'].create(child_vals)
+
+            # Create Holds for children to reserve
+            hold_vals = {
+                'name': self.name,
+                'child_id': child_comp.id,
+                'type': self.type,
+                'expiration_date': self.hold_expiration_date,
+                'primary_owner': self.primary_owner,
+                'secondary_owner': self.secondary_owner,
+                'no_money_yield_rate': self.no_money_yield_rate,
+                'yield_rate': self.yield_rate,
+                'channel': '',
```

Comment:
Add as well channel and source code as fields in the wizard, like you did for the others. Don't forget to add them in the view.